ios/BT/API/Model/ExposureConfiguration.swift

```diff
@@ -15,7 +15,7 @@ struct ExposureConfiguration: Codable {
 extension ExposureConfiguration {
 
   static var placeholder: ExposureConfiguration = {
-    ExposureConfiguration(minimumRiskScore: 1,
+    ExposureConfiguration(minimumRiskScore: 0,
```

Comment:
does this mean that we will only be receiving notifications on the js side that have at least a riskScore of 1?

Reply:
New exposures will trigger a push notification and be stored in Realm on the native side; so when they are fetched from the JS side they will reflect this exposure configuration